File: gluecodium/src/test/resources/smoke/defaults_const/output/android/com/example/smoke/ConstantDefaults.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
import com.example.fire.SomeStruct;
public final class ConstantDefaults {
    @NonNull
    public SomeStruct field1;
    @NonNull
    public SomeStruct field2;
    public ConstantDefaults() {
        this.field1 = com.example.fire.StructConstants.DUMMY;
        this.field2 = com.example.fire.StructConstants.DUMMY4;
    }
}

File: gluecodium/src/test/resources/smoke/defaults_const/output/android-kotlin/com/example/smoke/ConstantDefaults.kt
/*

 *
 */

@file:JvmName("ConstantDefaultsExtensions")

package com.example.smoke

import com.example.fire.SomeStruct

class ConstantDefaults {
    @JvmField var field1: SomeStruct
    @JvmField var field2: SomeStruct



    constructor() {
        this.field1 = com.example.fire.StructConstants.DUMMY
        this.field2 = com.example.fire.StructConstants.DUMMY4
    }





}

